_source/_posts/2020-11-24-spring-boot-okta.adoc

```diff
@@ -28,6 +30,8 @@ To show you how much fun it is, I https://youtu.be/dURkaz0kBSU[created a screenc
 </div>
 ++++
 
+**NOTE**: This video puts your settings in `src/main/resources/application.properties`. We've since changed the default behavior to use [spring-dotenv](https://github.com/okta-samples/okta-spring-boot-sample/pull/22).
```

Comment:
```suggestion
**NOTE**: This video puts your settings in `src/main/resources/application.properties`. We've since changed the default behavior to use https://github.com/okta-samples/okta-spring-boot-sample/pull/22[spring-dotenv].
```